docs/code-style.md

```diff
@@ -0,0 +1,333 @@
+# Code Style
+
+---
+
+**NOTE**:
+
+Most of the information in this document is meant to help keep the code looking
+similar and predictable while being worked on by different people. It isn't
+meant to be rigid and static, but a suggestion. Thus, rule of thumb:
+
+1. use your own best judgement;
+2. try to follow style of code already in place;
+3. notify others of issues and possible improvements.
+
+---
+
+## Comments (general)
+
+There usually isn't a formal way of writing comments. Most common are simple
+line comments, not block ones. However, few tags, due to familiarity with JSDoc,
+can be found outside these `/** */` blocks:
+
+- `@debt` is usually some future issue or pull request to repay with better code
+  solution be it refactor or replacement;
+- `@deprecated` is usually a comment that gives an explanation and alternative;
+- `@see` is usually followed by URL or similar reference for further info.
+
+## Style Sheets (coding/design)
+
+Over the course of the codebase there has been a revolving door of different
+(S)CSS and CSS-in-JS systems. This is an attempt to give a general overview of
+how they (can) coexist.
+
+### Previous Different systems
+
+There has been a mix of different style systems in use in the past. Some
+include:
+
+- _Styled Components_
+- SCSS provided by _Create React App_ (_CRA_), not including modules
+- _Bootstrap_ styles
+- global overrides for the most basic HTML tags
```

Comment:
Given that we're deleting all this in the redesign branch... do we need this section?

Reply:
A historical overview as an intro. Maybe just a tad of "here is what we got away from" before someone recommends Bootrstrap again a year from now :)

Anyway, different systems are in use now as well, the need of co-exist is there, just the named changed.